docs/topics/http/middleware.txt

```diff
@@ -292,8 +292,7 @@ Upgrading pre-Django 1.10-style middleware
 
 Django provides ``django.utils.deprecation.MiddlewareMixin`` to ease creating
 middleware classes that are compatible with both :setting:`MIDDLEWARE` and the
-old ``MIDDLEWARE_CLASSES``. All middleware classes included with Django
-are compatible with both settings.
```

Comment:
IMO this sentence is still true. I don't think that we need to remove it due to this documentation change.